README.md

```
@@ -157,60 +170,177 @@ Add this string to your `<head>` html-tag
 ```
-This helper uses a CDN-url for the widget bundle and supports three options:
 
-- **version** — version of the Uploadcare widget. Default is "3.x".
-- **bundle** — bundle name. Available names are "full", "default", "api", "ie8" and "lang.en".
-               Default bundle is "full" — a full bundle with built-in jQuery.
-               More info about bundles [here](https://uploadcare.com/docs/uploads/file-uploader/#bundles).
+This helper uses a CDN-url for the File Uploader and supports these options:
+
+- **version** — version of the Uploadcare File Uploader. Default is "v1".
+- **solution** — solution type. Available names are "regular", "inline", "minimal".
+               Default is "regular". More info about solutions [here](https://uploadcare.com/docs/file-uploader/solutions/).
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)